wpimath/src/main/java/edu/wpi/first/math/controller/ArmFeedforward.java

```diff
@@ -219,18 +193,10 @@ public Voltage calculate(Angle currentAngle, AngularVelocity currentVelocity) {
    * @param nextVelocity The next velocity setpoint.
    * @return The computed feedforward in volts.
    */
-  public Voltage calculate(
-      Angle currentAngle, AngularVelocity currentVelocity, AngularVelocity nextVelocity) {
-    return Volts.of(
-        ArmFeedforwardJNI.calculate(
-            ks,
-            kv,
-            ka,
-            kg,
-            currentAngle.in(Radians),
-            currentVelocity.in(RadiansPerSecond),
-            nextVelocity.in(RadiansPerSecond),
-            m_dt));
+  public double calculateWithVelocities(
```

Comment:
Since these are doubles now, the docs need to describe the units.

Reply:
like this?